index.html

```diff
@@ -0,0 +1,233 @@
+                <li class="nav-item"><a class="sub-item" href="#News">News</a></li>
+              </ul>
+              <a href="index.html"><button class="nav-works">@TRIC</button></a>
+            </nav>
+          </div>
+        </div>
+      </div>
+      <!--popup-->
+      <nav class="navmobile" id="navmobile">
+        <div class="btn-wrapper">
+            <button class="close-btn" onclick="closeMobileMenu()">
+              <img src="./image/close_mobile_menu.png" alt="closebtn">
+            </button>
+        </div>
+        <div class="logo" onclick="closeMobileMenu()">
+          <a href="home.html"><img class = "logo-img mobilelog" src="./image/mylogo.png" alt="the royal dev logo"/></a>
+        </div>
+        <ul class="navigation-item">
+          <li class="nav-mobile" onclick="closeMobileMenu()"><a class="mobile-item" href="about.html">About</a></li>
+          <li class="nav-mobile" onclick="closeMobileMenu()"><a class="mobile-item" href="#Program">Program</a></li>
+          <li class="nav-mobile" onclick="closeMobileMenu()"><a class="mobile-item" href="#Join">Join</a></li>
+          <li class="nav-mobile" onclick="closeMobileMenu()"><a class="mobile-item" href="#Sponsor">Sponsor</a></li>
+          <li class="nav-mobile" onclick="closeMobileMenu()"><a class="mobile-item" href="#News">News</a></li>
+        </ul>
+        <button class="nav-works nav-mobile" onclick="closeMobileMenu()"> @TRIC</button>
+      </nav>
+          <!--popup ends-->
+      <div class="intro-content">
+        <div class="intro-tittle">
+          <h1 class="intro1">"@TRIC Delivered To You!!"</h1>
+          <h2 class="intro2">
+            A NEW ERA OF CORPORATE VALUE CHAIN INNOVATION AND R&D LEADERS LIVEWORX 2023
+          </h2>
+        </div>
+        <div class="text-summary">
+          <p class="intro-text">
+            @THE ROYAL INNOVATION CENTER LiveWorx 2023 will bring together thousands
+            of corporate leaders across the innovation value chain in Kenya for four days to strengthen
+            the connections and fuel the conversations that contribute to a thriving ecosystem. From
+            ideation to commercialization- we'll tackle the new art of serious innovation.
+          </p>
+        </div>
+        <div class="vedate">
+          <h4 class="date">2023.03.08(WED)~11(SAT)</h4>
+          <p class="venue">
+            @ The Kenyatta International Convention Centre(KICC), Nairobi-Kenya
+          </p>
+        </div>
+      </div>
+    </header>
+
+    <main class="main">
+      <!--Main Program -->
+      <section class="program">
+        <div class="program-info">
+          <div class="pro-heading">
+            <h3 class="pro-text">Main Program</h3>
+            <div class="underline-orange"></div>
+            <hr class="line" />
+          </div>
+          <div class="procard">
+            <div class="program-container">
```

Comment:
Great job using <section> and <main> element. But let's use <article> for our program cards to make it more semantic.